packages/main/test/pages/TableSelection.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
```

Comment:
Please add a section in the Table.sample.html as well. It is a nice feature to showcase.

Reply:
Done